.github/workflows/unstable-release.yml

```diff
@@ -30,6 +30,9 @@ jobs:
         
       - name: Build packages
         run: pnpm build
-        
+
+      - name: Setup npmrc
```

Comment:
Can you use the secrets as we do for the regular release flow?

```
        env:
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
          NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
```

Reply:
Does this work? `NPM_TOKEN=... npm publish`

Because that's how I understand standard publishing to work - afaik you need to have an `.npmrc` like I've created here.

The difference between this and the example you're referencing is that it uses `changesets/action` which I assume has support for passing `NPM_TOKEN` as env var?

https://github.com/changesets/action#with-publishing